Changelog.md

```diff
@@ -2,6 +2,119 @@
 
 ### Gazebo Physics 6.x.x (202X-XX-XX)
 
+### Gazebo Physics 6.0.0 (2022-09-21)
+
+1. Improved documentation and instructions
+    * [Pull request #421](https://github.com/gazebosim/gz-physics/pull/421)
+    * [Pull request #419](https://github.com/gazebosim/gz-physics/pull/419)
+    * [Pull request #422](https://github.com/gazebosim/gz-physics/pull/422)
+    * [Pull request #417](https://github.com/gazebosim/gz-physics/pull/417)
+    * [Pull request #416](https://github.com/gazebosim/gz-physics/pull/416)
+    * [Pull request #316](https://github.com/gazebosim/gz-physics/pull/316)
+    * [Pull request #312](https://github.com/gazebosim/gz-physics/pull/312)
+    * [Pull request #401](https://github.com/gazebosim/gz-physics/pull/401)
+
+1. Added common tests
+    * [Pull request #414](https://github.com/gazebosim/gz-physics/pull/414)
+    * [Pull request #412](https://github.com/gazebosim/gz-physics/pull/412)
+    * [Pull request #411](https://github.com/gazebosim/gz-physics/pull/411)
+    * [Pull request #410](https://github.com/gazebosim/gz-physics/pull/410)
+    * [Pull request #409](https://github.com/gazebosim/gz-physics/pull/409)
+    * [Pull request #408](https://github.com/gazebosim/gz-physics/pull/408)
+    * [Pull request #407](https://github.com/gazebosim/gz-physics/pull/407)
+    * [Pull request #364](https://github.com/gazebosim/gz-physics/pull/364)
+    * [Pull request #362](https://github.com/gazebosim/gz-physics/pull/362)
+    * [Pull request #360](https://github.com/gazebosim/gz-physics/pull/360)
+    * [Pull request #363](https://github.com/gazebosim/gz-physics/pull/363)
+    * [Pull request #357](https://github.com/gazebosim/gz-physics/pull/357)
+
+1. Forward ports
```

Comment:
We can leave out forward ports and version bumps from the changelog

Reply:
Removed